keps/sig-apps/2307-job-tracking-without-lingering-pods/README.md

```diff
@@ -666,7 +666,7 @@ Yes, see [Deprecation](#deprecation) for the full plan.
       - Before 1.26: Observe the behavior in pods.
       - After 1.26: Based on metric `job_terminated_pod_tracking_finalizer`
     - Mitigations:
-      Before 1.26, disable `JobTrackingWithFinalizers`.
+      Update to latest patch versions for supported Kubernetes versions.
```

Comment:
I would just add the link to 119833 as part of this item. It's basically the same problem, just a little different trigger.

Reply:
Do you also want the finished jobs do not remove finalizers section removed?